.goreleaser.yml

```diff
@@ -1,11 +1,21 @@
 # Documentation: https://goreleaser.com/customization/
+# yaml-language-server: $schema=https://goreleaser.com/static/schema-pro.json
 version: 2
 project_name: step
 
+variables:
+  packageName: step-cli
+  packageRelease: 1 # Manually update release: in the nfpm section to match this value if you change this
```

Comment:
What's the reason for not using "{{ .Var.packageRelease }}" in the nfpm section?

Reply:
It will default to -1 and you only need to set that if we need to manually change the release and the nfpm section doesn't support variables.